otterdog/eclipse-tractusx.jsonnet

```diff
@@ -998,10 +1010,38 @@ orgs.newOrg('eclipse-tractusx') {
       ],
     },
     orgs.newRepo('ssi-authority-schema-registry') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      description: "SSI Authority & Schema Registry",
       private_vulnerability_reporting_enabled: true,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
is this necessary? we will wont to enforce read permissions by default on all repos which will be quite some work, but at least do not set back to write for existing ones.

If a workflow needs additional permissions you can specify that in the workflow itselt instead of just granting write tokens by default to all workflows. The same below.

Reply:
you're right, I removed them